example/src/App.tsx

```diff
@@ -111,23 +134,94 @@ export default class App extends React.Component {
     this._loadFontsAsync();
   }
 
+  renderScreen = (key: string, Screen: any) => {
+    return () => (
+      <Example key={key} title={key}>
+        <Screen />
+      </Example>
+    );
+  };
+
   render() {
+    const Drawer = createDrawerNavigator();
+
     if (this.state.fontsLoaded) {
       return (
         <Provider theme={DefaultTheme}>
-          <ScrollView style={{ flex: 1 }}>
-            {Object.entries(ROUTES).map(([key, Screen]) => {
-              return (
-                <Example key={key} title={key}>
-                  <Screen />
-                </Example>
-              );
-            })}
-          </ScrollView>
+          <SafeAreaProvider>
+            <SafeAreaView style={{ flex: 1 }}>
+              <NavigationContainer>
+                <Drawer.Navigator
+                  drawerContentOptions={{
+                    activeTintColor: "rgba(90, 69, 255, 1)",
+                  }}
+                  initialRouteName="Avatar"
+                >
+                  {Object.entries(ROUTES).map(([key, Screen]) => {
+                    return (
+                      <Drawer.Screen
+                        key={key}
+                        name={key}
+                        component={this.renderScreen(key, Screen)}
+                      />
+                    );
+                  })}
+                </Drawer.Navigator>
+              </NavigationContainer>
+            </SafeAreaView>
+          </SafeAreaProvider>
         </Provider>
       );
     } else {
       return <AppLoading />;
     }
   }
 }
+
+const exampleStyles = StyleSheet.create({
+  mainParent: {
+    width: screenWidth,
```

Comment:
do you need to include these? shouldn't this just work without it since a view's implicit width is 100%?

is there a difference between this and making the width 100%?

Reply:
From my testing, web misbehaved when using percentages (inconsistent with mobile), so I opted to use this approach.

Reply:
Scratch that, works fine everywhere except for images. Images do not appear when set using percentages (on web I mean). Can remove it everywhere except with images.

Reply:
Managed to remove them completely. Ignore all that